--- a/src/configure.ac
+++ b/src/configure.ac
@@ -857,23 +857,23 @@
     AC_MSG_ERROR([sed not found])
 fi

-AC_PATH_PROG(PS, ps, "none")
-if test $PS = "none"
-then
-    AC_MSG_ERROR([ps not found])
-fi
-
-AC_PATH_PROG(KILL, kill, "none")
-if test $KILL = "none"
-then
-    AC_MSG_ERROR([kill not found])
-fi
-
-AC_PATH_PROG(WHOAMI, whoami, "none")
-if test $WHOAMI = "none"
-then
-    AC_MSG_ERROR([whoami not found])
-fi
+# AC_PATH_PROG(PS, ps, "none")
+#if test $PS = "none"
+#then
+#    AC_MSG_ERROR([ps not found])
+#fi
+
+#AC_PATH_PROG(KILL, kill, "none")
+#if test $KILL = "none"
+#then
+#    AC_MSG_ERROR([kill not found])
+#fi
+
+#AC_PATH_PROG(WHOAMI, whoami, "none")
+#if test $WHOAMI = "none"
+#then
+#    AC_MSG_ERROR([whoami not found])
+#fi

 AC_PATH_PROG(AWK, awk, "none")
 if test $AWK = "none"
@@ -902,11 +902,11 @@
     fi
 fi

-AC_PATH_PROG(PIDOF, pidof, "none", $SPATH)
-if test $PIDOF = "none"
-then
-    AC_MSG_ERROR([pidof not found])
-fi
+#AC_PATH_PROG(PIDOF, pidof, "none", $SPATH)
+#if test $PIDOF = "none"
+#then
+#    AC_MSG_ERROR([pidof not found])
+#fi

 AC_PATH_PROG(IPCS, ipcs, "none")
 if test $IPCS = "none"
@@ -914,27 +914,27 @@
     AC_MSG_ERROR([ipcs not found])
 fi

-AC_PATH_PROG(FUSER, fuser, "none", $SPATH)
-if test $FUSER = "none"
-then
-    AC_MSG_ERROR([fuser not found])
-fi
-
-AC_PATH_PROGS(YAPPS, yapps yapps2, "none", $SPATH)
-if test $YAPPS = "none"
-then
-    AC_MSG_ERROR([yapps/yapps2 and not found])
-fi
-AC_SUBST(YAPPS)
+#AC_PATH_PROG(FUSER, fuser, "none", $SPATH)
+#if test $FUSER = "none"
+#then
+#    AC_MSG_ERROR([fuser not found])
+#fi
+
+#AC_PATH_PROGS(YAPPS, yapps yapps2, "none", $SPATH)
+#if test $YAPPS = "none"
+#then
+#    AC_MSG_ERROR([yapps/yapps2 and not found])
+#fi
+#AC_SUBST(YAPPS)

 # MANDB empty is handled in doc/Submakefile
 AC_PATH_PROG(MANDB, mandb, "")

-AC_PATH_PROG(INTLTOOL_EXTRACT, intltool-extract, "none", $SPATH)
-if test $INTLTOOL_EXTRACT = "none"
-then
-    AC_MSG_ERROR([intltool-extract not found])
-fi
+#AC_PATH_PROG(INTLTOOL_EXTRACT, intltool-extract, "none", $SPATH)
+#if test $INTLTOOL_EXTRACT = "none"
+#then
+#    AC_MSG_ERROR([intltool-extract not found])
+#fi

 AC_ARG_WITH(rmmod,
     AS_HELP_STRING(
@@ -978,12 +978,12 @@
             INSMOD="$EMC2_BIN_DIR/linuxcnc_module_helper insert"
     ])

-AC_PATH_PROGS(YAPPS, [yapps yapps2])
-if test -z "$YAPPS"; then
-  AC_MSG_ERROR([yapps is required to build LinuxCNC
-install with "sudo apt-get install yapps2"])
-fi
-AC_SUBST(YAPPS)
+#AC_PATH_PROGS(YAPPS, [yapps yapps2])
+#if test -z "$YAPPS"; then
+#  AC_MSG_ERROR([yapps is required to build LinuxCNC
+#install with "sudo apt-get install yapps2"])
+#fi
+#AC_SUBST(YAPPS)

 ##############################################################################
 # Subsection 3.7 - check for programs needed to build documentation          #
 